### PR TITLE
MM-27832 - Cypress/E2E: Channels > Change Channel using Channel Switcher

### PR DESCRIPTION
**Pull request by reflog**
#### Summary

Cypress/E2E: Automate backlogs - Channels > Change Channel using Channel Switcher (1 test case)

#### Ticket Link
Automates MM-T2031
https://mattermost.atlassian.net/browse/MM-27832

### CODE REVIEW

File: e2e/cypress/integration/channel/channel_switcher_spec.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            cy.apiCreateChannel(testTeam.id, 'channel-b', 'SwitchChannel B', 'O');
+            cy.apiCreateChannel(testTeam.id, 'channel-c', 'SwitchChannel C', 'O');
+
+            // # Login as test user and go to town square
+            cy.apiLogin(user);
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+    });
+
+    it('MM-T2031 - should switch channels by keyboard', () => {
```
Comment (josephbaylon) [inline e2e/cypress/integration/channel/channel_switcher_spec.js:31]: Hi @reflog , we usually have 1-to-1 mapping of our TM4J test steps to cypress test cases for accurate reporting. Please assign the test cases associated with the test TM4J test steps, such that `MM-T2031_<step number>`. 

If you deem 1-to-1 mapping is not achievable, please comment on the ones you combined so we can update the TM4J test case to reflect the same.

Thanks.

```suggestion
    it('MM-T2031_1 - should switch channels by keyboard', () => {
```

File: e2e/cypress/integration/channel/channel_switcher_spec.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
```
Comment (josephbaylon) [inline e2e/cypress/integration/channel/channel_switcher_spec.js:10]: Please remove `Stage: prod` for now until we promote test to stable.

```suggestion
```

**Comment by josephbaylon** [comment]
Hi @reflog, I've consolidated steps 1 & 2 in [MM-T2031](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T2031) per your enumeration. However, I don't see step 4 in the cypress test. Please add if not included. Thanks.

**Comment by reflog** [comment]
@josephbaylon oops. missed last step. added now.

**Review by ali-farooq0 — APPROVED** [review]
Neat 👍

**Pull request merged**